### PR TITLE
Support accessing shared files

### PR DESCRIPTION
**Pull request by dasheck0**
The current version of this project only supports accessing files and directories, which are owned. However the one drive api supports also accessing shared files. THis is easily down by changing the base route from `me/drive/items` to `users/$name/drive/items`. This is basically what this PR is all about. Please review and notify me if any changes are required.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+root = true
```
Comment (dkatavic) [inline .editorconfig]: thanks for adding `.editorconfig` :)

File: lib/items/listChildren.js [hunk 2]
```diff
@@ -7,30 +8,33 @@ var request = require('request-promise');
...
  *
  * @return {Array} object of children items
  */
 
 function listChildren(params) {
 
-  if (!params.accessToken) {
-    throw new Error("Missing params.accessToken");
-  }
+    if (!params.accessToken) {
```
Comment (dkatavic) [inline lib/items/listChildren.js]: Can you please reindent this file with 2 spaces instead of 4?

File: .editorconfig (hunk 1 above)
Comment (dasheck0) [reply .editorconfig]: No problem ;)

**Comment by dasheck0** [comment]
I reindented the malicious file and updated the pull request. After merging can you publish the npm module to npmjs.com? Thank you

**Comment by dkatavic** [comment]
I will publish the package after merging. 

ps. no worries about changelog, If I get the time later, I will update the changelog 👍

**Comment by dasheck0** [comment]
I updated the changelog. If there is anything more to do. Please let me know.

**Comment by dkatavic** [comment]
@dasheck0 Thanks. I am merging and publishing new package

**Pull request merged**